go/vt/sqlparser/ast_funcs.go

```diff
@@ -1118,6 +1118,19 @@ func (ty ExplainType) ToString() string {
 	}
 }
 
+func (sel SelectIntoType) ToString() string {
+	switch sel {
+	case IntoOutfile:
+		return IntoOutfileStr
+	case IntoOutfileS3:
+		return IntoOutfileS3Str
+	case IntoDumpfile:
+		return IntoDumpfileStr
+	default:
+		return "Unknown Select Into Type"
+	}
```

Comment:
I cant return an error here because this method is called from format which does not return any error.

Reply:
Actually, this statement would never run because it is an enum and the values are set in sql.y. THe default has to added so that the function declaration is complete. Is there some other way I should do this?